### DISCUSSION TITLE
How to make neovim LSP detect config and c?

### DISCUSSION THREAD

**Discussion by ll931217** [qutebrowser/qutebrowser#8294] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/2c2a0188-4ff4-472e-bb66-c83890e3c346)

As seen in the image above, the LSP (pyright) is showing these errors for `config` and `c`. I tried searching in qutebrowser source code for those 2 functions but couldn't find anything. Does anyone have a solution since I couldn't find anything on this here and reddit?

**Comment by The-Compiler** [comment, marked as answer]
See https://github.com/qutebrowser/qutebrowser/blob/main/doc%2Fhelp%2Fconfiguring.asciidoc#avoiding-flake8-errors - I'm assuming something similar should work too.

**Reply by ll931217** [reply]
Yes that seems to help with the errors, but doesn't seem to have LSP hints

**Reply by The-Compiler** [reply]
"LSP hints"?

**Reply by ll931217** [reply]
![image](https://github.com/user-attachments/assets/500dbef8-c8c2-4261-be0b-708356602648)
Yeah something like this, powered by nvim-cmp

**Comment by panv-00** [comment]
with basedpyright lsp, i use:

config = config  # type: ignore

config.load_autoconfig()
config.source("themes/tokyonight.py")
config.set("content.javascript.clipboard", "access")
